### PR TITLE
feat: add input amount and loader component

### PR DESCRIPTION
**Pull request by luizstacio**
closes: #179 
- Also, add a loader component with responsive support.

### CODE REVIEW

File: design-system/react/src/components/InputAmount/InputAmount.test.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    const maxBtn = screen.getByLabelText('Max');
+    expect(maxBtn).toBeInTheDocument();
+    expect(screen.getByPlaceholderText('0.00')).toBeInTheDocument();
+
+    await user.click(maxBtn);
+    expect(screen.getByDisplayValue('14.563943834')).toBeInTheDocument();
+  });
+
+  it('should unable to input more than balance', async () => {
```
Comment (LuizAsFight) [inline design-system/react/src/components/InputAmount/InputAmount.test.tsx:71]:
```suggestion
  it('should be able to input more than balance', async () => {
```

File: design-system/react/src/components/InputAmount/InputAmount.test.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    render(<Content balance={bn(500000000)} />);
+    expect(screen.getByText('Balance: 0.5')).toBeInTheDocument();
+
+    const input = screen.getByLabelText(FIELD_NAME);
+    fireEvent.input(input, { target: { value: '0.6' } });
+
+    expect(input.getAttribute('value')).toBe('0.6');
+  });
+});
```
Comment (LuizAsFight) [inline design-system/react/src/components/InputAmount/InputAmount.test.tsx:80]: can we add tests to make sure this inputs are working correctly ?
```
  units,
  hiddenBalance,
  hiddenMaxButton,
```

File: design-system/react/src/components/InputAmount/InputAmount.test.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    />
+  );
+}
+
+describe('InputAmount', () => {
+  const onChange = jest.fn();
+
+  it('a11y', async () => {
+    await testA11y(<Content onChange={onChange} />);
+  });
```
Comment (LuizAsFight) [inline design-system/react/src/components/InputAmount/InputAmount.test.tsx:39]: can we also `a11y` the Loader component ?

File: design-system/react/src/utils/css.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export function getCSSValue(value?: string | number | undefined) {
```
Comment (LuizAsFight) [inline design-system/react/src/utils/css.ts:1]: I would change the function name to `toCSSValue` , which imples a conversion/parse style...

`getCSSValue` sounds like a function that would take an input element, and give me computed CSS value from it

if you think it makes sense lets change

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

**Comment by LuizAsFight** [comment]
also found a functional problem around the `leadingZero` behavior:

https://user-images.githubusercontent.com/8636507/205770605-e7b8bcee-e595-4615-85fa-8638b4628a31.mov


notice that after you start to repeatedly prefix the value with `0(zero)`
- value gets cut visually (but I'm using input with no action), should not be cut. why are we using `visibility` to control if buttons appear or not? it hides the element, but maintains the space in the DOM. I think that's the problem
- prefixxed zeros should be ignored, but they're being shown. they only start to be hidden after I start to change the real number (after prefixed zeros). Zeros start to reduce one by one.
- let's add a test for it: any prefixed zero should be considered invalid and not rendered

**Comment by LuizAsFight** [comment]
in the `Input Amount` stories, would also be nice to add the `amount` value outside of the `InputAmount` component, to visually make sure it is the same of the value inside the component

File: design-system/react/src/components/InputAmount/InputAmount.test.tsx (hunk 3 above)
Comment (luizstacio) [reply design-system/react/src/components/InputAmount/InputAmount.test.tsx:39]: We already have

**Review by LuizAsFight — APPROVED** [review]
